webpack.common.config.js

```diff
@@ -2,7 +2,7 @@ var path      = require('path');
 var EMPTY     = 'widget/empty';
 var TARGET_JS = path.resolve(__dirname, 'target/js/');
 var SHARED_JS = TARGET_JS + '/shared';
-var VENDOR = path.resolve(__dirname, 'packages/@okta/courage/src/vendor');
```

Comment:
Interesting - why isn't this caught in lint (no-undefined-vars)?

Reply:
Seems like a few of our rules weren't being applied to files located at the root. I've updated this PR to now include `webpack.*` files.